### PR TITLE
Remove inline styles from the current conditions component

### LINKED ISSUE #198: Address inline styles in the current conditions weather block
Once we have our USWDS theme in place, then we can start to figure out how we want to go about applying styling to various components. For now, I just put them right in the Twig template. That's probably not where they should live forever, though, so let's remember to go clean that up.

Introduced in #196.

### PR DESCRIPTION
**Pull request by greg-does-weather**
## What's wrong?

When it was originally created, the styles for the current conditions component were hardcoded right into the component code. Instead, it should use USWDS utility classes.

## How does this PR fix it? 

- Swaps out font size styles for USWDS utility classes
- Swaps out margins for utility classes
- Pulls the styling that's unique to the component into its own sass file

This component is probably going to need some more styling work in order to be *right*, but this is a step in the right direction.

image

Closes #198.

## Types of changes

- [x] 🐛 Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the code style of this project.

### CODE REVIEW

File: web/themes/new_weather_theme/assets/sass/components/_current-conditions.scss [hunk 1]
```diff
@@ -0,0 +1,16 @@
+.weather-gov-current-conditions {
+  & .icon svg {
+    height: 64px;
```
Comment (claire-skies-noaa) [inline web/themes/new_weather_theme/assets/sass/components/_current-conditions.scss:3]:
```suggestion
    @include u-height(8);
```

File: web/themes/new_weather_theme/assets/sass/components/_current-conditions.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+    & path.st0 {
+      fill: black;
+    }
+  }
+
+  & .item {
+    & svg {
+      height: 16px;
```
Comment (claire-skies-noaa) [inline web/themes/new_weather_theme/assets/sass/components/_current-conditions.scss:12]:
```suggestion
      @include u-height(2);
```

File: web/themes/new_weather_theme/assets/sass/components/_current-conditions.scss [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+      width: 16px;
```
Comment (claire-skies-noaa) [inline web/themes/new_weather_theme/assets/sass/components/_current-conditions.scss:13]:
```suggestion
      @include u-height(2);
```

File: web/themes/new_weather_theme/assets/sass/components/_current-conditions.scss [hunk 4]
```diff
@@ -0,0 +1,16 @@
+.weather-gov-current-conditions {
+  & .icon svg {
+    height: 64px;
+
+    & path.st0 {
+      fill: black;
```
Comment (claire-skies-noaa) [inline web/themes/new_weather_theme/assets/sass/components/_current-conditions.scss:6]:
```suggestion
      fill: color('black');
```

**Review by claire-skies-noaa — CHANGES_REQUESTED** [review]
This is great, Greg! Thanks for creating the component sass. Let's start using USWDS utility mixins instead of hardcoding values. I've included code suggestions. As long you add 
`@use "uswds-core" as *;` at the top of the SASS partial, you'll have access to all utility mixins and tokens. 

References:

[Height and width mixins](https://designsystem.digital.gov/utilities/height-and-width/#utility-mixins)
[The color tokens and mixins](https://designsystem.digital.gov/design-tokens/color/theme-tokens/#using-theme-color-tokens-2)

**Comment by greg-does-weather** [comment]
@claire-skies-noaa Thank you! I'm going to have to get into the habit of keeping the USWDS reference open more often.

**Pull request merged**